flowzone.yml

```diff
@@ -2369,6 +2394,41 @@ jobs:
           path: ${{ runner.temp }}/docs.tar.zst
           retention-days: 90
 
+  npm_sbom:
+    name: Generate SBOM for NPM
+    runs-on: ${{ fromJSON(inputs.runs_on) }}
+    needs:
+      - is_npm
+      - npm_test
+      - versioned_source
+    if: ${{ needs.is_npm.outputs.npm == 'true' && needs.is_npm.outputs.npm_sbom == 'true' }}
+    <<: *customWorkingDirectory
+    steps:
+      - *getGitHubAppToken
+      - *checkoutVersionedSha
+      - *createLocalRefs
+      - *setupNode
+
+      - run: npm install
+
+      - name: Generate SBOM
+        run: |
+          npx @cyclonedx/cyclonedx-npm --output-file=${{ runner.temp }}/bom.xml
+
+      - <<: *publishSBOMArtifacts
+        with:
+          name: gh-release-sbom-npm
+          path: ${{ runner.temp }}/bom.xml
```

Comment:
I believe the artifacts inside the zip need to have unique names or they will overwrite each other when being attached to the release. See the output [here](https://github.com/product-os/flowzone/releases/tag/untagged-16e8c096dae978954854).

Reply:
makes sense

Reply:
It's fixed :) https://github.com/product-os/flowzone/releases/tag/untagged-e866e755aabd0a215bb2